test/dataframerow.jl

```diff
@@ -267,9 +267,11 @@ end
 end
 
 @testset "convert, copy and merge" begin
-    df = DataFrame(a=[1, missing], b=[2.0, 3.0])
+    df = DataFrame(a=[1, missing, missing], b=[2.0, 3.0, 0.0])
     dfr = DataFrameRow(df, 1, :)
-
+    nt = first(Tables.namedtupleiterator(df))
+    @test copy(dfr) === nt === NamedTuple{(:a, :b),Tuple{Union{Missing, Int64},Float64}}((1, 2.0))
```

Comment:
```suggestion
    @test copy(dfr) === nt === NamedTuple{(:a, :b),Tuple{Union{Missing, Int},Float64}}((1, 2.0))
```
correcting as I guess without it the PR will fail on 32-bit Julia